storey/table.py

```diff
@@ -17,9 +19,12 @@ class Table:
     :param storage: Storage driver
     :param partitioned_by_key: Whether that data is partitioned by the key or not, based on this indication storage drivers
      can optimize writes. Defaults to True.
+    :param flush_interval_secs: interval in seconds to flush the cache to storage
```

Comment:
Currently disabled when 0 is passed. Needs docu and maybe a clearer way of setting this.